models/heirbrant2023boekbazaar/metadata.yaml

```diff
@@ -0,0 +1,34 @@
+title: Boekbazaar Ontology
+acronym:
+issued: 2023
+modified:
+contributor:
+ - https://www.linkedin.com/in/joachim-heirbrant-532aa1233/
+ - https://dblp.org/pid/251/5991
+ - https://dblp.org/pid/95/6356
+keyword:
+ - digital platform ontology
+ - digital platform
+ - marketplace
+theme: Class H - Social Sciences
+editorialNote: Ontology created during an Ontology Engineering lecture by students who wanted to remain anonymous. The ontology was build in a different editor tool and imported to Visual Paradigm, the current diagrammation may not exacly match the original one.
```

Comment:
This note seems wrong. The author is identified by his/her thesis.

Reply:
You are right. Fixed.